crates/iota/src/client_ptb/ptb.rs

```diff
@@ -45,9 +45,9 @@ pub struct PTB {
     pub display: HashSet<DisplayOption>,
 }
 
-pub struct PTBPreview<'a> {
```

Comment:
Just curious, can you elaborate on the ownership change please?

Reply:
It was just an unnecessary lifetime when this data can be owned. Otherwise I would have to propagate it upward in the containing enums